app/cache.py

```diff
@@ -50,7 +50,7 @@ def create_cache():
     try:
         playlist_json = requests.get(
             f'{XOS_API_ENDPOINT}playlists/{XOS_PLAYLIST_ID}/',
-            timeout=5,
+            timeout=50,
```

Comment:
Curling the `JSON` endpoint from the balena device I only see ~7 second response. So shall we increase it to `15` rather than `50`?

`curl -O https://xos.acmi.net.au/api/playlists/43/?format=json`